{
  "uid" : "5d6857b5d5264b2e",
  "name" : "Проверка что авторизованный пользователь может выйти из аккаунта",
  "fullName" : "tests.test_workspace_page.test_workspace_page.TestWorkspacePage#test_auth_user_logout",
  "historyId" : "ef004c13c91f13fae735f1a09c2da0ef",
  "time" : {
    "start" : 1713882593647,
    "stop" : 1713882593955,
    "duration" : 308
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\n  (Session info: chrome-headless-shell=123.0.6312.122); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0x5598d1bde863 <unknown>\n#1 0x5598d18d48c6 <unknown>\n#2 0x5598d18d9e75 <unknown>\n#3 0x5598d18dbbce <unknown>\n#4 0x5598d18dbc5c <unknown>\n#5 0x5598d19211c2 <unknown>\n#6 0x5598d19143a1 <unknown>\n#7 0x5598d19415a2 <unknown>\n#8 0x5598d1913e98 <unknown>\n#9 0x5598d194176e <unknown>\n#10 0x5598d195fc19 <unknown>\n#11 0x5598d1941343 <unknown>\n#12 0x5598d1912593 <unknown>\n#13 0x5598d1912f5e <unknown>\n#14 0x5598d1ba284b <unknown>\n#15 0x5598d1ba67a5 <unknown>\n#16 0x5598d1b90571 <unknown>\n#17 0x5598d1ba7332 <unknown>\n#18 0x5598d1b7587f <unknown>\n#19 0x5598d1bcd728 <unknown>\n#20 0x5598d1bcd8fb <unknown>\n#21 0x5598d1bdd9b4 <unknown>\n#22 0x7fcc67894ac3 <unknown>",
  "statusTrace" : "self = <tests.test_workspace_page.test_workspace_page.TestWorkspacePage object at 0x7f9be9d20d40>\nauth_user = <pages.login_page.LoginPage object at 0x7f9be9d5c710>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"4dac27316ce6c45239726986a4dbd328\")>\n\n    @allure.title(\"Проверка что авторизованный пользователь может выйти из аккаунта\")\n    @pytest.mark.regress\n    def test_auth_user_logout(self, auth_user, driver):\n        page = SignUpPage(driver)\n>       page.click_button_avatar()\n\ntests/test_workspace_page/test_workspace_page.py:50: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/signup_page.py:93: in click_button_avatar\n    self.element_is_clickable(element).click()\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py:94: in click\n    self._execute(Command.CLICK_ELEMENT)\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute\n    return self._parent.execute(command, params)\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:347: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f9be9d5fd10>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"stale element reference\",\"message\":\"stale element reference: stale elemen...\\\\n#20 0x5598d1bcd8fb \\\\u003Cunknown>\\\\n#21 0x5598d1bdd9b4 \\\\u003Cunknown>\\\\n#22 0x7fcc67894ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\nE         (Session info: chrome-headless-shell=123.0.6312.122); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nE       Stacktrace:\nE       #0 0x5598d1bde863 <unknown>\nE       #1 0x5598d18d48c6 <unknown>\nE       #2 0x5598d18d9e75 <unknown>\nE       #3 0x5598d18dbbce <unknown>\nE       #4 0x5598d18dbc5c <unknown>\nE       #5 0x5598d19211c2 <unknown>\nE       #6 0x5598d19143a1 <unknown>\nE       #7 0x5598d19415a2 <unknown>\nE       #8 0x5598d1913e98 <unknown>\nE       #9 0x5598d194176e <unknown>\nE       #10 0x5598d195fc19 <unknown>\nE       #11 0x5598d1941343 <unknown>\nE       #12 0x5598d1912593 <unknown>\nE       #13 0x5598d1912f5e <unknown>\nE       #14 0x5598d1ba284b <unknown>\nE       #15 0x5598d1ba67a5 <unknown>\nE       #16 0x5598d1b90571 <unknown>\nE       #17 0x5598d1ba7332 <unknown>\nE       #18 0x5598d1b7587f <unknown>\nE       #19 0x5598d1bcd728 <unknown>\nE       #20 0x5598d1bcd8fb <unknown>\nE       #21 0x5598d1bdd9b4 <unknown>\nE       #22 0x7fcc67894ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: StaleElementReferenceException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1713882588361,
      "stop" : 1713882588642,
      "duration" : 281
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1713881392683,
      "stop" : 1713881392683,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "auth_user",
    "time" : {
      "start" : 1713882588642,
      "stop" : 1713882593647,
      "duration" : 5005
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Проверка видимости кнопки 'Принимаю все' в сообщении о принятии файлов cookie",
      "time" : {
        "start" : 1713882591885,
        "stop" : 1713882591915,
        "duration" : 30
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Нажать кнопку 'Войти'",
      "time" : {
        "start" : 1713882592081,
        "stop" : 1713882592130,
        "duration" : 49
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Ждать обновление страницы",
      "time" : {
        "start" : 1713882592130,
        "stop" : 1713882593647,
        "duration" : 1517
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 3,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1713881392665,
      "stop" : 1713881392683,
      "duration" : 18
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\n  (Session info: chrome-headless-shell=123.0.6312.122); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0x5598d1bde863 <unknown>\n#1 0x5598d18d48c6 <unknown>\n#2 0x5598d18d9e75 <unknown>\n#3 0x5598d18dbbce <unknown>\n#4 0x5598d18dbc5c <unknown>\n#5 0x5598d19211c2 <unknown>\n#6 0x5598d19143a1 <unknown>\n#7 0x5598d19415a2 <unknown>\n#8 0x5598d1913e98 <unknown>\n#9 0x5598d194176e <unknown>\n#10 0x5598d195fc19 <unknown>\n#11 0x5598d1941343 <unknown>\n#12 0x5598d1912593 <unknown>\n#13 0x5598d1912f5e <unknown>\n#14 0x5598d1ba284b <unknown>\n#15 0x5598d1ba67a5 <unknown>\n#16 0x5598d1b90571 <unknown>\n#17 0x5598d1ba7332 <unknown>\n#18 0x5598d1b7587f <unknown>\n#19 0x5598d1bcd728 <unknown>\n#20 0x5598d1bcd8fb <unknown>\n#21 0x5598d1bdd9b4 <unknown>\n#22 0x7fcc67894ac3 <unknown>",
    "statusTrace" : "self = <tests.test_workspace_page.test_workspace_page.TestWorkspacePage object at 0x7f9be9d20d40>\nauth_user = <pages.login_page.LoginPage object at 0x7f9be9d5c710>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"4dac27316ce6c45239726986a4dbd328\")>\n\n    @allure.title(\"Проверка что авторизованный пользователь может выйти из аккаунта\")\n    @pytest.mark.regress\n    def test_auth_user_logout(self, auth_user, driver):\n        page = SignUpPage(driver)\n>       page.click_button_avatar()\n\ntests/test_workspace_page/test_workspace_page.py:50: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/signup_page.py:93: in click_button_avatar\n    self.element_is_clickable(element).click()\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py:94: in click\n    self._execute(Command.CLICK_ELEMENT)\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute\n    return self._parent.execute(command, params)\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:347: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f9be9d5fd10>\nresponse = {'status': 404, 'value': '{\"value\":{\"error\":\"stale element reference\",\"message\":\"stale element reference: stale elemen...\\\\n#20 0x5598d1bcd8fb \\\\u003Cunknown>\\\\n#21 0x5598d1bdd9b4 \\\\u003Cunknown>\\\\n#22 0x7fcc67894ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\nE         (Session info: chrome-headless-shell=123.0.6312.122); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nE       Stacktrace:\nE       #0 0x5598d1bde863 <unknown>\nE       #1 0x5598d18d48c6 <unknown>\nE       #2 0x5598d18d9e75 <unknown>\nE       #3 0x5598d18dbbce <unknown>\nE       #4 0x5598d18dbc5c <unknown>\nE       #5 0x5598d19211c2 <unknown>\nE       #6 0x5598d19143a1 <unknown>\nE       #7 0x5598d19415a2 <unknown>\nE       #8 0x5598d1913e98 <unknown>\nE       #9 0x5598d194176e <unknown>\nE       #10 0x5598d195fc19 <unknown>\nE       #11 0x5598d1941343 <unknown>\nE       #12 0x5598d1912593 <unknown>\nE       #13 0x5598d1912f5e <unknown>\nE       #14 0x5598d1ba284b <unknown>\nE       #15 0x5598d1ba67a5 <unknown>\nE       #16 0x5598d1b90571 <unknown>\nE       #17 0x5598d1ba7332 <unknown>\nE       #18 0x5598d1b7587f <unknown>\nE       #19 0x5598d1bcd728 <unknown>\nE       #20 0x5598d1bcd8fb <unknown>\nE       #21 0x5598d1bdd9b4 <unknown>\nE       #22 0x7fcc67894ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: StaleElementReferenceException",
    "steps" : [ {
      "name" : "Нажать на иконку аватара",
      "time" : {
        "start" : 1713882593647,
        "stop" : 1713882593954,
        "duration" : 307
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame\n  (Session info: chrome-headless-shell=123.0.6312.122); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception\nStacktrace:\n#0 0x5598d1bde863 <unknown>\n#1 0x5598d18d48c6 <unknown>\n#2 0x5598d18d9e75 <unknown>\n#3 0x5598d18dbbce <unknown>\n#4 0x5598d18dbc5c <unknown>\n#5 0x5598d19211c2 <unknown>\n#6 0x5598d19143a1 <unknown>\n#7 0x5598d19415a2 <unknown>\n#8 0x5598d1913e98 <unknown>\n#9 0x5598d194176e <unknown>\n#10 0x5598d195fc19 <unknown>\n#11 0x5598d1941343 <unknown>\n#12 0x5598d1912593 <unknown>\n#13 0x5598d1912f5e <unknown>\n#14 0x5598d1ba284b <unknown>\n#15 0x5598d1ba67a5 <unknown>\n#16 0x5598d1b90571 <unknown>\n#17 0x5598d1ba7332 <unknown>\n#18 0x5598d1b7587f <unknown>\n#19 0x5598d1bcd728 <unknown>\n#20 0x5598d1bcd8fb <unknown>\n#21 0x5598d1bdd9b4 <unknown>\n#22 0x7fcc67894ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/PulseWave/PulseWave/pages/signup_page.py\", line 93, in click_button_avatar\n    self.element_is_clickable(element).click()\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "617504287d184669",
      "name" : "Screenshot",
      "source" : "617504287d184669.png",
      "type" : "image/png",
      "size" : 56733
    }, {
      "uid" : "17efb3e95c681efd",
      "name" : "HTML source",
      "source" : "17efb3e95c681efd.html",
      "type" : "text/html",
      "size" : 72617
    } ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1713882594127,
      "stop" : 1713882594185,
      "duration" : 58
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Тестирование страницы 'Рабочие пространства'"
  }, {
    "name" : "tag",
    "value" : "regress"
  }, {
    "name" : "parentSuite",
    "value" : "tests.test_workspace_page"
  }, {
    "name" : "suite",
    "value" : "test_workspace_page"
  }, {
    "name" : "subSuite",
    "value" : "TestWorkspacePage"
  }, {
    "name" : "host",
    "value" : "fv-az695-415"
  }, {
    "name" : "thread",
    "value" : "2345-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_workspace_page.test_workspace_page"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "browser",
    "value" : "'chrome'"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 1,
        "skipped" : 0,
        "passed" : 1,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "fd106e7038ddf492",
        "reportUrl" : "https://Nat754.github.io/PulseWave/208//#testresult/fd106e7038ddf492",
        "status" : "passed",
        "time" : {
          "start" : 1713879771347,
          "stop" : 1713879771758,
          "duration" : 411
        }
      } ]
    },
    "tags" : [ "regress" ]
  },
  "source" : "5d6857b5d5264b2e.json",
  "parameterValues" : [ "'chrome'" ]
}